src/main/java/techcourse/myblog/service/LoginService.java

```diff
@@ -0,0 +1,41 @@
+package techcourse.myblog.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import techcourse.myblog.controller.dto.LoginDTO;
+import techcourse.myblog.exception.LoginFailException;
+import techcourse.myblog.exception.UserNotExistException;
+import techcourse.myblog.model.User;
+import techcourse.myblog.repository.UserRepository;
+
+@Service
+public class LoginService {
+    private UserRepository userRepository;
+
+    @Autowired
```

Comment:
단일 생성자인 경우, `@Autowired`가 없어도 됩니다.

> If a class only declares a single constructor to begin with, it will always be used, even if not annotated.

```suggestion

```